test/doc/algo/equal.cpp

```diff
@@ -0,0 +1,23 @@
+#include <eve/module/core.hpp>
+#include <eve/algo.hpp>
+#include <iostream>
+#include <vector>
+#include "detail/print.hpp"
+
+int main()
+{
+  std::vector<float> r1(5), r2(5);
+  auto vr1 = eve::views::iota_with_step(0.5f, 2.0f, 5);
+  eve::algo::copy(vr1, r1);
+  eve::algo::copy(vr1, r2);
+  r2[2] = 100.0f;
+  std::cout << " <- std::vector<float> r1(5), r2(5);\n";
+  std::cout << " <- auto vr1 = eve::views::iota_with_step(0.5f, 2.0f, 5);\n";
+  std::cout << " <- eve::algo::copy(vr1, r1);\n";
```

Comment:
Can we interleave display of variables and their contents ?
I have no idea what is inside vr1